src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -383,7 +383,7 @@ public class HttpProtocolTests : HttpClientTestBase
         [InlineData("HTTP/1.1 200", 200, "")] // This test data requires the fix in .NET Framework 4.7.3
         [InlineData("HTTP/1.1 200 O\tK", 200, "O\tK")]
         [InlineData("HTTP/1.1 200 O    \t\t  \t\t\t\t  \t K", 200, "O    \t\t  \t\t\t\t  \t K")]
-        // TODO #24713: The following pass on Windows but fail on CurlHandler on Linux.
+        // Only CurlHandler will trim the '\t' at the end and causing failure.
```

Comment:
Not sure how to deal with this test data. Creating a new method/splitting test cases like below are expensive for only one test data.

I added comment as a documentation for the behavior difference.